src/templates/haml/user/reality_check.html.haml

```diff
@@ -43,7 +43,7 @@
     - }
   %p
     %label= l('Reality Check Interval:')
-    %input{:type => 'text' :interval => 1 :size => '6'}
+    %input{:type => 'text' :interval => 1 :size => '6' :id =>'realityDuration'}
```

Comment:
Hi boris ;

Thanks for the suggestion. Have though about it as well. But input type displays like below 
![image](https://cloud.githubusercontent.com/assets/12234030/11387361/a05c2504-9364-11e5-969d-9c9f46f7e0ad.png)

With an up and down arrow option. We though it does not looks good that way. :)